### DISCUSSION TITLE
关于自训练模型的使用

### DISCUSSION THREAD

**Discussion by syy-love** [PaddlePaddle/PaddleSpeech#2208] [Q&A] [answered]
自己训练的fastpeech2_cnndecoder_csmsc_onnx 想在流式tts中用 如何操作呀 我把model里的文件替换 启动时它会更新覆盖

**Comment by yt605155624** [comment, marked as answer]
可以在配置文件指定模型路径
https://github.com/PaddlePaddle/PaddleSpeech/blob/b0b3222f9a967c2fe1b3f944b177d467a11c2146/demos/streaming_tts_server/conf/tts_online_application.yaml#L68

加载本地模型逻辑参考 
https://github.com/PaddlePaddle/PaddleSpeech/blob/b0b3222f9a967c2fe1b3f944b177d467a11c2146/paddlespeech/server/engine/tts/online/onnx/tts_engine.py#L116
